internal/stdlib/os/exec.go

```diff
@@ -20,9 +21,8 @@ var ErrProcessDone = errors.New("os: process already finished")
 // Process stores the information about a process created by StartProcess.
 type Process struct {
 	Pid    int
-	handle uintptr      // handle is accessed atomically on Windows
-	isdone uint32       // process has been successfully waited on, non zero if true
-	sigMu  sync.RWMutex // avoid race between wait and signal
```

Comment:
the mutex wasn't used anywhere and you deleted it or was there another reason?

Reply:
Correct yea, was only used in a Linux codepath.